virttest/libvirt_xml/accessors.py

```diff
@@ -786,7 +786,12 @@ def __call__(self, value):
             # Allow other classes to generate parent structure
             parent = self.xmltreefile().find(self.parent_xpath)
             if parent is None:
-                raise xcepts.LibvirtXMLNotFoundError
+                # Create parent xpath if not exists
+                self.xmltreefile().create_by_xpath(self.parent_xpath)
```

Comment:
This is super class for all libvirtXML, can we test more types of xml other than VMCPUXML ?

Reply:
@chunfuwen  That's a good idea, but unfortunately, we do not have that kind of test cases for now. Would you like to give some other suggestions about the tests? Thanks.

Reply:
something similar  like above VMCPUXML, for example:create xml such as disk, network

Reply:
@chunfuwen  Again, I don't think there is that kind of tests now. Could you help provide the test cases you mentioned? I have tried, but cannot find them by myself. Thank you very much.